.github/workflows/release.yml

```diff
@@ -0,0 +1,59 @@
+on:
+  release:
+    types: [published]
+
+name: Build and publish app release
+
+jobs:
+  build_and_publish:
+    name: Rust project
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        target: [ aarch64-unknown-linux-gnu, armv7-unknown-linux-gnueabihf, i686-unknown-linux-gnu, powerpc64le-unknown-linux-gnu, x86_64-unknown-linux-gnu, s390x-unknown-linux-gnu, x86_64-pc-windows-gnu, i686-pc-windows-gnu, i686-unknown-linux-musl, mips-unknown-linux-gnu, mips64-unknown-linux-gnuabi64, mips64el-unknown-linux-gnuabi64, mipsel-unknown-linux-gnu, powerpc-unknown-linux-gnu, powerpc64-unknown-linux-gnu, arm-unknown-linux-gnueabi, x86_64-unknown-linux-musl ]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: set release version
+        shell: bash
+        run: python3 -m pip install toml-cli && toml set --toml-path Cargo.toml package.version ${{ github.ref_name }}
+
+      - name: Install rust
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+          profile: minimal
+          override: true
+          target: ${{ matrix.target }}
+
+      - name: Build target
+        uses: actions-rs/cargo@v1
+        with:
+          use-cross: true
+          command: build
+          args: --release --target ${{ matrix.target }}
+
+      - name: Package linux
+        shell: bash
+        if: ${{ !contains(matrix.target, 'windows') }}
+        run: |
+          mv target/${{ matrix.target }}/release/gsltctrl gsltctrl-${{ matrix.target }}
+
+      - name: Package windows
+        shell: bash
+        if: ${{ contains(matrix.target, 'windows') }}
+        run: |
+          mv target/${{ matrix.target }}/release/gsltctrl.exe gsltctrl-${{ matrix.target }}.exe
+
+      - name: show checksum
+        shell: bash
+        run: md5sum gsltctrl-*
```

Comment:
Also, are those hashes saved to files and added to the release?

Reply:
You are right, i've changed it to sha256sum :) 
The purpose of this step is not to add them later, but so that anyone can check if the file that is attached to the release (and that is downloaded) is really the one, that is built in the gh action.